### ISSUE TITLE
Update to list-item 1.0.x

### ISSUE THREAD

**Issue by AndersDJohnson** [jonschlinkert/bullets#1]
As soon as latest jonschlinkert/list-item#1 1.0.x is published to npm, we should update the dependency version here.
Not sure how much to bump this package's version semantically - @jonschlinkert ?

**Comment by jonschlinkert** [comment]
sound good, I added you as a collab. feel free to publish 1.0.1 of list-item and update this. I think bumping  `bullets` to `0.2.0` would be make sense. or I can do it in the next couple of days. thanks!

**Comment by AndersDJohnson** [comment]
@jonschlinkert It won't let me publish to npm - was I really added as an npm owner per jonschlinkert/markdown-utils#2?

**Comment by jonschlinkert** [comment]
hmm, what's your npm username? perhaps I typed it wrong...

**Comment by jonschlinkert** [comment]
you're right though, I just checked and it doesn't look like I added it. let me try again brb

**Comment by jonschlinkert** [comment]
nvm it's not showing via command line, but when you go to https://www.npmjs.com/package/markdown-utils, you're listed there as a maintainer. let me know if you're still having issues, we can file a ticket if necessary

**Comment by AndersDJohnson** [comment]
@jonschlinkert Thanks, `markdown-utils` looks good, just need `bullets`.
